GameData/RP-1/Localization/zh-cn.cfg

```diff
@@ -0,0 +1,165 @@
+		#rp0_Leaders_CanRemoveAfter = Can Remove After
+		#rp0_Leaders_RetiresIn = Retires In
+		#rp0_Leaders_RetiresOn = Retires On
+		#rp0_Leaders_RetiresAfter = Retires After
+		#rp0_Leaders_Remove_ConfirmWithCost = 你确定要开除这个领导人吗? 这将会花费:\n<<1>><<2>>
+		#rp0_Leaders_Remove_Confirm = 你确定要开除这个领导人吗?<<1>>
+		#rp0_Leaders_Deactivates = 这个领导人一旦被开除，就不能再被重新任命到任何职位上！
+		#rp0_Leaders_Deactivates_WithCooldown = 该领导人在被开除超过<<1>>后才能再次担任职务！
+		#rp0_Leaders_Appoint_CannotAfford = 承担不起雇佣
+		#rp0_Leaders_Remove_CannotAfford = 承担不起开除
+		#rp0_Leaders_HiringRequirements = 雇佣要求
+		#rp0_Leaders_HiringRequirements_Unmet = 不满足雇佣要求
+		#rp0_Leaders_LeadersUnlocked = 以下的领导人是可用的:\n
+		#rp0_Leaders_LeadersUnlockedTitle = 解锁了领导人!
+		#rp0_Leaders_UnlocksLeader = 解锁以下的领导人:\n
+
+		// Leader effects
+		#rp0_Leaders_Effect_ResearchRateModifier = <<1>>  <<2>> 的研究速度
+		#rp0_Leaders_Effect_IntegrationRateModifier = 从<<1>> 到 <<2>> 的整合拼装速度
+		#rp0_Leaders_Effect_CurrencyModifier = <<1>> <<2>> <<3>>
+		// overrides
+		#rp0_Leaders_Effect_CurrencyModifierIntegrationRate = <<1>> 到载具拼装速度
+
+		// General strings
+		#rp0_Generic_DiskSpace_B = <<1>> B
+		#rp0_Generic_DiskSpace_kB = <<1>> kB
+		#rp0_Generic_DiskSpace_MB = <<1>> MB
+		#rp0_Generic_Percent_Positive = +<<1>>%
+		#rp0_Generic_Percent_Negative = -<<1>>%
+		#rp0_Generic_Value_Positive = +<<1>>
+		#rp0_Generic_Value_Negative = -<<1>>
+		#rp0_Generic_Cost = 花费: <<1>>
+		#rp0_Generic_Reward = 奖励: <<1>>
+
+		#rp0_Currency_Confidence = 信用点
+		#rp0_Currency_Time = 时间
+		#rp0_Currency_Rate = 速率
+		#rp0_Currency_Format_Rate = <<1>>x
+		
+
+		// KSP hardcodes using #autoLOC for Facility context menus.
+		#autoLOC_rp0_FacilityContextMenu_VAB = 从管理中心的窗口修改发射综合体 
+		#autoLOC_rp0_FacilityContextMenu_SPH = 从管理中心的窗口修改机库 
+		#rp0_FacilityContextMenu_RnD_ResearcherLimit = 工程师限制: <<1>>
+		#rp0_FacilityContextMenu_RnD_ResearcherLimit_unlimited = 无限制
+		#autoLOC_rp0_FacilityContextMenu_AC_RnR = <<1>> R&R 时间
+		#autoLOC_rp0_FacilityContextMenu_AC_Training = <<1>> 训练时间
+		#rp0_FacilityContextMenu_TechGate = 不能升级这个设施，他需要 <<1>>
+		#rp0_FacilityContextMenu_AlreadyUpgrading =已经在升级了
+		#rp0_FacilityContextMenu_UpgradeCostTime = 预计花费资金/日: <<1>>\n完成于 <<2>>
+		#rp0_FacilityContextMenu_UpgradeCostDate = 预计花费资金/日: <<1>>\n完成于 <<2>>
+		#rp0_FacilityContextMenu_UpgradeStart = 建设开始!
+		#rp0_FacilityContextMenu_UpgradeInProgress = 正在进行建设中!
+
+		// Unlock Credit
+		#rp0_UnlockCredit_NodeInfo = 解锁的贷款额度: <sprite=\"CurrencySpriteAsset\" name=\"Funds\" tint=1><<1>>\nIncluding Parents: <sprite=\"CurrencySpriteAsset\" name=\"Funds\" tint=1><<2>>
+		#rp0_UnlockCredit_CostAfterCredit = 除去贷款后价格: √<<1>>
+
+
+		// Currency widget tooltips
+		#rp0_Widgets_Science_Tooltip = 已经获得的科技点: <<1>>\n解锁的贷款总额:<sprite=\"CurrencySpriteAsset\" name=\"Funds\" tint=1><<2>>
+		#rp0_Widgets_Reputation_Tooltip = 声望控制着津贴水平，同时津贴水平也跟着时间的推移而减少\n通过合同以及提前完成项目获得声望,\n而错过项目的最终期限则会损失声望.\n\n最低津贴: <<1>> (at 0 Rep)\n可能的最高津贴: <<2>> (at <<3>> Rep)\n当前的年度津贴: <<4>>\n(最小值/最大值随时间变化)\n每天损失 <<5>> Rep  (<<6>>/年)
```

Comment:
[85a4ed9](https://github.com/KSP-RO/RP-1/commit/85a4ed9191141e6adddaf6e421a3ff34876783a6)，close meaning ,no need to change Chinese translation.